mapserver/deploy.sh

```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
-set -euo pipefail
-
-cd $(dirname $0)
-
-# To be invoked from circleci
-
-if [ $# -lt 1 ]; then
-  echo "Usage: $0 DOCKER_TAG"
-	exit 1
-fi
-
-TAG=${1/\//_}
-
-MAPCACHE_TAG=instedd/planwise-mapserver:mapcache-$TAG
-MAPSERVER_TAG=instedd/planwise-mapserver:mapserver-$TAG
```

Comment:
Oh, I see why you added the tag suffix above. I think in this case the `TAG` was not meant to be a versioning tag, but a content tag. Ie. at some point we would generate the mapserver image with some regional TIFFs included. But I don't think we've used them in that way for a very long time.

Reply:
Interesting.

I see [in Docker Hub](https://hub.docker.com/r/instedd/planwise-mapserver/tags) we have some tag `kenya-mapserver-stable`, but I also see `mapcache-latest`, `mapcache-master` & `mapcache-stable` (created a few years ago).

I mean, I really am not sure about what these things are or do, so I'll trust your judgment here. I'm just replicating the CI we had.

Reply:
Oh, wait. I didn't realize that we didn't have a different repo for mapcache. That's what the tag is being used for. I'd go for a different Docker repo though.

Reply:
Agree on that. I'll leave that for later, though.